src/index.html

```diff
@@ -11,7 +11,177 @@
+          <label for="aad">Full date of birth:</label>
+          <input
+          type="date"
+          name="date of birth"
+          id="date-of-birth"
+          required
+          >
+        </div>
+
+        <div class="lastInput">
+          <label for="terms-condition">I accept the term of the agreement</label>
+          <input
+          type="checkbox"
+          name="attribute"
+          id="terms-condition"
+          required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field block">
+        <legend>Registration</legend>
+        <div class="input">
+          <label for="aae">E-mail:</label>
+          <input
+          type="email"
+          name="attribute"
+          placeholder="email@example.com"
+          required
+          >
+        </div>
+        <div class="lastInput">
+          <label for="aaf">Password:</label>
+          <input
+          type="password"
+          name="attribute"
+          autocomplete="off"
+          required
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field block">
+        <legend>An interesting fact about you!</legend>
+        <div class="input">
+          <label for="aag">Do you love cats?</label>
+          <label for="yes">
+            <input
+            type="radio"
+            id="yes"
+            name="radio"
+            checked
+            >
+            Yes
+          </label>
+          <label for="no">
+            <input
+            type="radio"
+            id="no"
+            name="radio"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="input">
+          <label for="aah">What is your favorite color?</label>
+          <input
+          type="color"
+          name="attribute"
+          required
+          >
+        </div>
+
+        <div class="input">
+          <label for="aba">What time do you go to bed?</label>
+          <input
+          type="time"
+          name="attribute"
+          required
+          >
+        </div>
+
+        <div class="input">
+          <label for="abb">What are your favorite brand of cars?</label>
+          <select name="car"
+          multiple
+          required>
+            <option value="Audi">Audi</option>
+            <option value="BMW">BMW</option>
+            <option value="carExpert">Lada</option>
+            <option value="none" disabled></option>
+          </select>
+        </div>
+
+        <div class="lastInput">
+          <label for="abc">How do you rate our work?</label>
+          <input
+          type="range"
+          required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form-field block">
+        <legend>Additional info:</legend>
+        <div class="input">
+          <label for="abd">Comments:</label>
+          <textarea
+          name="comments"
+          maxlength="300"
+          autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            maxlength="300"
            autocomplete="off"
        ></textarea>
```
Let's fix this problem
![image](https://user-images.githubusercontent.com/96409563/222509847-b2550cea-55ac-4fa8-9b87-d03d9b6ee456.png)
![image](https://user-images.githubusercontent.com/96409563/222509905-837c8bde-cfbe-47c1-83c1-2bb032f877fc.png)